### DISCUSSION TITLE
Date Range CSV format

### DISCUSSION THREAD

**Discussion by stupidaddfly** [vzhd1701/csv2notion#24] [Q&A]
I'm merging my old database with a new one and I've converted my xlsx on my own to CSV but I couldn't manage to get date ranges correctly and I wanted to confirm whether I did it right.
In Notion have one column called Work Time with date property that uses date range to determine time I spent working.
I Excel I have the same column name, with date range between date 1 and date 2 being like this "YYYY-MM-DDTHH:MM:SSSZ/YYYY-MM-DDTHH:MM:SSSZ", or on live example "2023-03-03T10:02:000Z/2023-03-03T19:13:000Z" yet when I import merge CSV the date range isn't recognized. Could I ask for help, how should I format my date range so it works properly?
Thanks in advance!

**Comment by vzhd1701** [comment]
Use comma to separate dates. Also put your range column in quotes
```
name,period
test1,"2023-03-03T10:02:000Z,2023-03-03T19:13:000Z"
test2,"2023-04-03T10:02:000Z,2023-04-03T19:13:000Z"
```
